src/Geometry.cpp

```diff
@@ -110,8 +77,8 @@ double Geometry::getXMin() {
  * @brief Return the maximum x-coordinate contained by the Geometry.
  * @return the maximum x-coordinate (cm)
  */
-double Geometry::getXMax() {
-  return _x_max;
+double Geometry::getMaxX() {
+  return _root_universe->getMaxX();
```

Comment:
The problem with storing it in the ``Geometry`` is that the user may change a ``Surface`` parametrically - hence changing the bounding box around various ``Cells``, ``Universes``, etc. under the hood. The ``Geometry`` will not know about this unless it traverses the CSG tree to find the appropriate bounding box. This allows geometric parameters to be varied with multiple simulations in a single Python script.